apps/studio/src/components/common/texteditor/TextEditor.vue

```diff
@@ -129,6 +130,11 @@ export default {
     },
   },
   methods: {
+    focusEditor() {
+      if(this.editor && this.autoFocus){
```

Comment:
I think this might need to be a level up for the cases where there are multiple editor tabs.

@azmy60 Are there changes that need to be made here to account for `tab.active`?

Reply:
Yes, this will steal focus from other components even though the text editor wasn't focused.

Right before the window is blurred, we should check if the editor is on focus. Once we reopen the window, the text editor knows if it should focus or not. These are the steps:

1. Before the window is blurred, we can use `document.activeElement` to see if text editor was focused? put this in a state (called `focusAfterWindowFocus` or smth like that 🤷)
2. After the window gains focus, we should call focus() if the state is true, and then switch the state to false